compiler/noirc_driver/src/lib.rs

```diff
@@ -334,6 +334,7 @@ pub fn compile_no_check(
     force_compile: bool,
 ) -> Result<CompiledProgram, RuntimeError> {
     let program = monomorphize(main_function, &context.def_interner);
+    println!("{}", program);
```

Comment:
Can you remove this println?

Reply:
I add this println so much I wonder if I should just make it a compiler flag